src/main/java/com/iexec/common/replicate/ReplicateStatusDetails.java

```diff
@@ -25,4 +37,11 @@ public ReplicateStatusDetails(long blockNumber) {
     public ReplicateStatusDetails(ReplicateStatusCause cause) {
         this.cause = cause;
     }
+
+    public ReplicateStatusDetails cropStdout() {
```

Comment:
I would `tail stdout` instead of `head stdout`

Reply:
Thanks for noticing it !